ngraph/frontend/paddlepaddle/src/op/hard_sigmoid.cpp

```diff
@@ -0,0 +1,35 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <ngraph/opsets/opset6.hpp>
+#include "hard_sigmoid.hpp"
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+
+NamedOutputs hard_sigmoid (const NodeContext& node) {
+    auto data = node.get_ng_input("X");
+    auto slope = node.get_attribute<float>("slope");
```

Comment:
Spec says that it is optional, how does it work? Should we also care about optional parameters or paddle paddle would return default here if nothing was set by user?

Reply:
@iknewnth, could have a lot ?

Reply:
The current implementation refers to  https://github.com/PaddlePaddle/Paddle2ONNX/blob/71c5b1688838b48fa2c227bce511bac08fb919da/paddle2onnx/op_mapper/activation.py#L102. I'll add the optional feature, thanks.